packages/site/site-nextjs/src/sitePreview/appRouter/sitePreviewRoute.ts

```diff
@@ -40,14 +40,19 @@ export async function sitePreviewRoute(request: NextRequest) {
  * @param options.skipDraftModeCheck Allows skipping the draft mode check, only required when called from middleware.ts (see https://github.com/vercel/next.js/issues/52080)
  * @return If SitePreview is active the current preview settings
  */
-export async function previewParams(options: { skipDraftModeCheck: boolean } = { skipDraftModeCheck: false }): Promise<SitePreviewParams | null> {
+export async function previewParams(options: { skipDraftModeCheck: boolean } = { skipDraftModeCheck: false }): Promise<PreviewParams | null> {
```

Comment:
Maybe we should move previewParams in to a separate file to "show" that it's now used for both previews. Same for verifyJwt. Maybe previewUtils.ts?